### ISSUE TITLE
Not working with Angular 2.0.0-rc.1

### ISSUE THREAD

**Issue by codeepic** [rangle/augury#416]
Hi,

I am running Angular 2 app (2.0.0-rc.1) with TS and SystemJS module loader on localhost and the Augury inspection tab doesn't show any information on my app. I am using Chrome (Version 50.0.2661.102 m). I checked the demo app (https://augury.angular.io/demo/) and the information in Augury tab is in there, but nothing for my app.

**Comment by bzums** [comment]
Same here, maybe it is not working with bundled/minified apps or so?

**Comment by sumitarora** [comment]
@codeepic @bzums you should be running the app in dev mode for Augury to work. If to enable Production mode Augury won't show you any details as production builds don't have any debug information.

**Comment by bzums** [comment]
My fault, restarting the browser after installing sometimes really helps!!1!11 😇

**Comment by codeepic** [comment]
Ok, same here. Restarted the browser and it's working now.

**Comment by sumitarora** [comment]
@codeepic can you close the issue? Thanks!!

**Comment by davutg** [comment]
Mine, really doesn't work.
I minify all js files for my development env. and use source-mappers.
Angular version is angular2@2.0.0-beta.2. 
I've installed Augury from chrome web store and it works for one another project. 
I'll put my problematic one to github soon :)

Index.html
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width">
    <title>Option Select Control</title>    
    <link rel="stylesheet" href="css/style.css">
    <script src="js/lib/angular2/es6-shim.min.js"></script>
    <script src="js/lib/angular2/system-polyfills.js"></script>
    <script src="js/lib/angular2/angular2-polyfills.js"></script>
    <script src="js/lib/angular2/system.src.js"></script>
    <script src="js/lib/angular2/Rx.js"></script>
    <script src="js/lib/angular2/angular2.dev.js"></script>
    <script src="js/lib/angular2/http.js"></script>
    <script src="js/lib/angular2/jquery.min.js"></script>
    <script>
      System.config({
        packages: { 
         js: {
            format: 'register',
            defaultExtension: 'js'
          }
        },
            bundles: {
              'js/whole.js': ['boot','angular2/platform/browser', "app.component"]
            }
          
      });
      System.import('boot')
        .then(null, console.error.bind(console));
    </script>
  </head>

<body>
  <my-app>Loading...</my-app>
</body>
</html>
```

**Here is my npm list for this project**
+-- **angular2@2.0.0-beta.2**
+-- del@2.2.2
| +-- globby@5.0.0
| | +-- array-union@1.0.2
| | +-- arrify@1.0.1
| | -- glob@7.1.1
| |   +-- fs.realpath@1.0.0
| |   +-- inflight@1.0.6
| |   | -- wrappy@1.0.2
| |   +-- once@1.4.0
| |   -- path-is-absolute@1.0.1
| +-- is-path-cwd@1.0.0
| +-- is-path-in-cwd@1.0.0
| | -- is-path-inside@1.0.0
| |   -- path-is-inside@1.0.2
| +-- object-assign@4.1.0
| +-- pify@2.3.0
| +-- pinkie-promise@2.0.1
| | -- pinkie@2.0.4
| -- rimraf@2.5.4
+-- es6-promise@3.3.1
+-- es6-shim@0.33.13
+-- gulp@3.9.1
| +-- archy@1.0.0
| +-- chalk@1.1.3
| | +-- ansi-styles@2.2.1
| | +-- escape-string-regexp@1.0.5
| | +-- has-ansi@2.0.0
| | | -- ansi-regex@2.0.0
| | +-- strip-ansi@3.0.1
| | -- supports-color@2.0.0
| +-- deprecated@0.0.1
| +-- gulp-util@3.0.7
| | +-- array-differ@1.0.0
| | +-- array-uniq@1.0.3
| | +-- beeper@1.1.0
| | +-- dateformat@1.0.12
| | | +-- get-stdin@4.0.1
| | | -- meow@3.7.0
| | |   +-- camelcase-keys@2.1.0
| | |   | -- camelcase@2.1.1
| | |   +-- loud-rejection@1.6.0
| | |   | +-- currently-unhandled@0.4.1
| | |   | | -- array-find-index@1.0.2
| | |   | -- signal-exit@3.0.1
| | |   +-- map-obj@1.0.1
| | |   +-- normalize-package-data@2.3.5
| | |   | +-- hosted-git-info@2.1.5
| | |   | +-- is-builtin-module@1.0.0
| | |   | | -- builtin-modules@1.1.1
| | |   | -- validate-npm-package-license@3.0.1
| | |   |   +-- spdx-correct@1.0.2
| | |   |   | -- spdx-license-ids@1.2.2
| | |   |   -- spdx-expression-parse@1.0.4
| | |   +-- read-pkg-up@1.0.1
| | |   | +-- find-up@1.1.2
| | |   | | -- path-exists@2.1.0
| | |   | -- read-pkg@1.1.0
| | |   |   +-- load-json-file@1.1.0
| | |   |   | -- parse-json@2.2.0
| | |   |   |   -- error-ex@1.3.0
| | |   |   |     -- is-arrayish@0.2.1
| | |   |   -- path-type@1.1.0
| | |   +-- redent@1.0.0
| | |   | +-- indent-string@2.1.0
| | |   | | -- repeating@2.0.1
| | |   | |   -- is-finite@1.0.2
| | |   | |     -- number-is-nan@1.0.1
| | |   | -- strip-indent@1.0.1
| | |   -- trim-newlines@1.0.0
| | +-- fancy-log@1.2.0
| | | -- time-stamp@1.0.1
| | +-- lodash._reescape@3.0.0
| | +-- lodash._reevaluate@3.0.0
| | +-- lodash._reinterpolate@3.0.0
| | +-- lodash.template@3.6.2
| | | +-- lodash._basecopy@3.0.1
| | | +-- lodash._basetostring@3.0.1
| | | +-- lodash._basevalues@3.0.0
| | | +-- lodash._isiterateecall@3.0.9
| | | +-- lodash.escape@3.2.0
| | | | -- lodash._root@3.0.1
| | | +-- lodash.keys@3.1.2
| | | | +-- lodash._getnative@3.9.1
| | | | +-- lodash.isarguments@3.1.0
| | | | -- lodash.isarray@3.0.4
| | | +-- lodash.restparam@3.6.1
| | | -- lodash.templatesettings@3.1.1
| | +-- multipipe@0.1.2
| | | -- duplexer2@0.0.2
| | |   -- readable-stream@1.1.14
| | +-- object-assign@3.0.0
| | +-- replace-ext@0.0.1
| | -- vinyl@0.5.3
| +-- interpret@1.0.1
| +-- liftoff@2.3.0
| | +-- extend@3.0.0
| | +-- findup-sync@0.4.3
| | | +-- detect-file@0.1.0
| | | | -- fs-exists-sync@0.1.0
| | | +-- is-glob@2.0.1
| | | | -- is-extglob@1.0.0
| | | +-- micromatch@2.3.11
| | | | +-- arr-diff@2.0.0
| | | | | -- arr-flatten@1.0.1
| | | | +-- array-unique@0.2.1
| | | | +-- braces@1.8.5
| | | | | +-- expand-range@1.8.2
| | | | | | -- fill-range@2.2.3
| | | | | |   +-- is-number@2.1.0
| | | | | |   +-- isobject@2.1.0
| | | | | |   | -- isarray@1.0.0
| | | | | |   -- randomatic@1.1.5
| | | | | +-- preserve@0.2.0
| | | | | -- repeat-element@1.1.2
| | | | +-- expand-brackets@0.1.5
| | | | | -- is-posix-bracket@0.1.1
| | | | +-- extglob@0.3.2
| | | | +-- filename-regex@2.0.0
| | | | +-- kind-of@3.0.4
| | | | | -- is-buffer@1.1.4
| | | | +-- normalize-path@2.0.1
| | | | +-- object.omit@2.0.1
| | | | | +-- for-own@0.1.4
| | | | | | -- for-in@0.1.6
| | | | | -- is-extendable@0.1.1
| | | | +-- parse-glob@3.0.4
| | | | | +-- glob-base@0.3.0
| | | | | | -- glob-parent@2.0.0
| | | | | -- is-dotfile@1.0.2
| | | | -- regex-cache@0.4.3
| | | |   +-- is-equal-shallow@0.1.3
| | | |   -- is-primitive@2.0.0
| | | -- resolve-dir@0.1.1
| | |   -- global-modules@0.2.3
| | |     +-- global-prefix@0.1.4
| | |     | +-- ini@1.3.4
| | |     | +-- osenv@0.1.3
| | |     | | -- os-tmpdir@1.0.2
| | |     | -- which@1.2.11
| | |     |   -- isexe@1.1.2
| | |     -- is-windows@0.2.0
| | +-- fined@1.0.2
| | | +-- expand-tilde@1.2.2
| | | +-- lodash.assignwith@4.2.0
| | | +-- lodash.isempty@4.4.0
| | | +-- lodash.pick@4.4.0
| | | -- parse-filepath@1.0.1
| | |   +-- is-absolute@0.2.6
| | |   | -- is-relative@0.2.1
| | |   |   -- is-unc-path@0.1.1
| | |   |     -- unc-path-regex@0.1.2
| | |   +-- map-cache@0.2.2
| | |   -- path-root@0.1.1
| | |     -- path-root-regex@0.1.2
| | +-- flagged-respawn@0.3.2
| | +-- lodash.isplainobject@4.0.6
| | +-- lodash.isstring@4.0.1
| | +-- lodash.mapvalues@4.6.0
| | +-- rechoir@0.6.2
| | -- resolve@1.1.7
| +-- minimist@1.2.0
| +-- orchestrator@0.3.8
| | +-- end-of-stream@0.1.5
| | | -- once@1.3.3
| | +-- sequencify@0.0.7
| | -- stream-consume@0.1.0
| +-- pretty-hrtime@1.0.3
| +-- semver@4.3.6
| +-- tildify@1.2.0
| | -- os-homedir@1.0.2
| +-- v8flags@2.0.11
| | -- user-home@1.1.1
| -- vinyl-fs@0.3.14
|   +-- defaults@1.0.3
|   +-- glob-stream@3.1.18
|   | +-- glob@4.5.3
|   | +-- glob2base@0.0.12
|   | | -- find-index@0.1.1
|   | +-- minimatch@2.0.10
|   | +-- ordered-read-streams@0.1.0
|   | +-- through2@0.6.5
|   | | -- readable-stream@1.0.34
|   | -- unique-stream@1.0.0
|   +-- glob-watcher@0.0.6
|   | -- gaze@0.5.2
|   |   -- globule@0.1.0
|   |     +-- glob@3.1.21
|   |     | +-- graceful-fs@1.2.3
|   |     | -- inherits@1.0.2
|   |     +-- lodash@1.0.2
|   |     -- minimatch@0.2.14
|   |       +-- lru-cache@2.7.3
|   |       -- sigmund@1.0.1
|   +-- graceful-fs@3.0.11
|   | -- natives@1.1.0
|   +-- mkdirp@0.5.1
|   | -- minimist@0.0.8
|   +-- strip-bom@1.0.0
|   | -- first-chunk-stream@1.0.0
|   +-- through2@0.6.5
|   | -- readable-stream@1.0.34
|   -- vinyl@0.4.6
|     -- clone@0.2.0
+-- gulp-concat@2.6.0
| +-- concat-with-sourcemaps@1.0.4
| -- through2@0.6.5
|   +-- readable-stream@1.0.34
|   | +-- core-util-is@1.0.2
|   | -- string_decoder@0.10.31
|   -- xtend@4.0.1
+-- gulp-minify@0.0.14
| +-- gulp-util@2.2.20
| | +-- chalk@0.5.1
| | | +-- ansi-styles@1.1.0
| | | +-- has-ansi@0.1.0
| | | | -- ansi-regex@0.2.1
| | | +-- strip-ansi@0.3.0
| | | -- supports-color@0.2.0
| | +-- lodash._reinterpolate@2.4.1
| | +-- lodash.template@2.4.1
| | | +-- lodash._escapestringchar@2.4.1
| | | +-- lodash.defaults@2.4.1
| | | | +-- lodash._objecttypes@2.4.1
| | | | -- lodash.keys@2.4.1
| | | +-- lodash.escape@2.4.1
| | | | +-- lodash._escapehtmlchar@2.4.1
| | | | | -- lodash._htmlescapes@2.4.1
| | | | -- lodash._reunescapedhtml@2.4.1
| | | |   -- lodash.keys@2.4.1
| | | +-- lodash.keys@2.4.1
| | | | +-- lodash._isnative@2.4.1
| | | | +-- lodash._shimkeys@2.4.1
| | | | -- lodash.isobject@2.4.1
| | | +-- lodash.templatesettings@2.4.1
| | | -- lodash.values@2.4.1
| | |   -- lodash.keys@2.4.1
| | +-- minimist@0.2.0
| | +-- through2@0.5.1
| | | -- xtend@3.0.0
| | -- vinyl@0.2.3
| +-- minimatch@3.0.3
| | -- brace-expansion@1.1.6
| |   +-- balanced-match@0.4.2
| |   -- concat-map@0.0.1
| +-- through2@0.4.2
| | +-- readable-stream@1.0.34
| | -- xtend@2.1.2
| |   -- object-keys@0.4.0
| -- uglify-js@2.7.4
|   +-- async@0.2.10
|   +-- uglify-to-browserify@1.0.2
|   -- yargs@3.10.0
|     +-- camelcase@1.2.1
|     +-- cliui@2.1.0
|     | +-- center-align@0.1.3
|     | | +-- align-text@0.1.4
|     | | | +-- longest@1.0.1
|     | | | -- repeat-string@1.6.1
|     | | -- lazy-cache@1.0.4
|     | +-- right-align@0.1.3
|     | -- wordwrap@0.0.2
|     +-- decamelize@1.2.0
|     -- window-size@0.1.0
+-- gulp-rename@1.2.2
+-- gulp-sourcemaps@1.9.0
| +-- acorn@4.0.3
| +-- convert-source-map@1.3.0
| +-- css@2.2.1
| | +-- inherits@2.0.3
| | +-- source-map@0.1.43
| | | -- amdefine@1.0.1
| | +-- source-map-resolve@0.3.1
| | | +-- atob@1.1.3
| | | +-- resolve-url@0.2.1
| | | -- source-map-url@0.3.0
| | -- urix@0.1.0
| +-- debug-fabulous@0.0.4
| | +-- debug@2.2.0
| | | -- ms@0.7.1
| | -- lazy-debug-legacy@0.0.1
| +-- detect-newline@2.1.0
| +-- graceful-fs@4.1.10
| +-- source-map@0.5.6
| +-- strip-bom@2.0.0
| | -- is-utf8@0.2.1
| +-- through2@2.0.1
| | -- readable-stream@2.0.6
| |   +-- isarray@1.0.0
| |   +-- process-nextick-args@1.0.7
| |   -- util-deprecate@1.0.2
| -- vinyl@1.2.0
|   +-- clone@1.0.2
|   -- clone-stats@0.0.1
+-- gulp-typescript@2.14.1
| +-- typescript@1.8.10
| -- vinyl-fs@2.4.4
|   +-- duplexify@3.5.0
|   | +-- end-of-stream@1.0.0
|   | | -- once@1.3.3
|   | +-- readable-stream@2.1.5
|   | | -- isarray@1.0.0
|   | -- stream-shift@1.0.0
|   +-- glob-stream@5.3.5
|   | +-- glob@5.0.15
|   | +-- glob-parent@3.0.1
|   | | +-- is-glob@3.1.0
|   | | | -- is-extglob@2.1.0
|   | | -- path-dirname@1.0.2
|   | +-- ordered-read-streams@0.3.0
|   | | -- is-stream@1.1.0
|   | +-- through2@0.6.5
|   | | -- readable-stream@1.0.34
|   | |   -- isarray@0.0.1
|   | +-- to-absolute-glob@0.1.1
|   | | -- extend-shallow@2.0.1
|   | -- unique-stream@2.2.1
|   |   -- json-stable-stringify@1.0.1
|   |     -- jsonify@0.0.0
|   +-- gulp-sourcemaps@1.6.0
|   +-- is-valid-glob@0.3.0
|   +-- lazystream@1.0.0
|   | -- readable-stream@2.1.5
|   |   -- isarray@1.0.0
|   +-- lodash.isequal@4.4.0
|   +-- merge-stream@1.0.0
|   | -- readable-stream@2.1.5
|   |   -- isarray@1.0.0
|   +-- readable-stream@2.1.5
|   | +-- buffer-shims@1.0.0
|   | -- isarray@1.0.0
|   +-- strip-bom-stream@1.0.0
|   +-- through2-filter@2.0.0
|   +-- vali-date@1.0.0
|   -- vinyl@1.2.0
+-- gulp-uglify@2.0.0
| +-- gulplog@1.0.0
| | -- glogg@1.0.0
| +-- has-gulplog@0.1.0
| | -- sparkles@1.0.0
| +-- lodash@4.16.6
| +-- make-error-cause@1.2.2
| | -- make-error@1.2.1
| +-- uglify-js@2.7.0
| +-- uglify-save-license@0.4.1
| -- vinyl-sourcemaps-apply@0.2.1
+-- gulp-webserver@0.9.1
| +-- connect@3.5.0
| | +-- finalhandler@0.5.0
| | | +-- on-finished@2.3.0
| | | | -- ee-first@1.1.1
| | | +-- statuses@1.3.0
| | | -- unpipe@1.0.0
| | +-- parseurl@1.3.1
| | -- utils-merge@1.0.0
| +-- connect-livereload@0.4.1
| +-- gulp-util@2.2.20
| | +-- chalk@0.5.1
| | | +-- ansi-styles@1.1.0
| | | +-- has-ansi@0.1.0
| | | | -- ansi-regex@0.2.1
| | | +-- strip-ansi@0.3.0
| | | -- supports-color@0.2.0
| | +-- lodash._reinterpolate@2.4.1
| | +-- lodash.template@2.4.1
| | | +-- lodash.escape@2.4.1
| | | +-- lodash.keys@2.4.1
| | | -- lodash.templatesettings@2.4.1
| | +-- minimist@0.2.0
| | -- vinyl@0.2.3
| +-- isarray@0.0.1
| +-- node.extend@1.1.6
| | -- is@3.2.0
| +-- open@0.0.5
| +-- proxy-middleware@0.5.1
| +-- serve-index@1.8.0
| | +-- accepts@1.3.3
| | | -- negotiator@0.6.1
| | +-- batch@0.5.3
| | +-- escape-html@1.0.3
| | +-- http-errors@1.5.0
| | | +-- inherits@2.0.1
| | | -- setprototypeof@1.0.1
| | -- mime-types@2.1.12
| |   -- mime-db@1.24.0
| +-- serve-static@1.11.1
| | +-- encodeurl@1.0.1
| | -- send@0.14.1
| |   +-- depd@1.1.0
| |   +-- destroy@1.0.4
| |   +-- etag@1.7.0
| |   +-- fresh@0.3.0
| |   +-- mime@1.3.4
| |   -- range-parser@1.2.0
| +-- through2@0.5.1
| | +-- readable-stream@1.0.34
| | -- xtend@3.0.0
| +-- tiny-lr@0.1.4
| | +-- body-parser@1.8.4
| | | +-- bytes@1.0.0
| | | +-- depd@0.4.5
| | | +-- iconv-lite@0.4.4
| | | +-- media-typer@0.3.0
| | | +-- on-finished@2.1.0
| | | | -- ee-first@1.0.5
| | | +-- qs@2.2.4
| | | +-- raw-body@1.3.0
| | | -- type-is@1.5.7
| | |   -- mime-types@2.0.14
| | |     -- mime-db@1.12.0
| | +-- debug@0.8.1
| | +-- faye-websocket@0.7.3
| | | -- websocket-driver@0.6.5
| | |   -- websocket-extensions@0.1.1
| | -- qs@2.2.5
| -- watch@0.11.0
+-- jquery@3.1.1
+-- reflect-metadata@0.1.2
+-- rxjs@5.0.0-beta.0
+-- systemjs@0.19.6
| +-- es6-module-loader@0.17.11
| -- when@3.7.7
+-- typescript@1.8.10
-- zone.js@0.5.10

**Comment by winnemucca** [comment]
Are there conditions for Augury to work?  I can get it fine on small projects I am starting with the cli but I cannot get this to work on on larger production apps.  ```This application is not an Angular application.
This application cannot be inspected using Augury.```.   Not sure what can cause this.

**Comment by JoeKahl** [comment]
I have similar problem.
src/index.ts has:
// enableProdMode();
This changed nothing.  Still cannot use Augury.
Seems like complex apps have this problem?

**Comment by davutg** [comment]
@JoeKahl  Are you using some other extensions? Such as AngularJS Inspector for Angular 1x. 
Give a try disabling other extensions but let Augry work. In which version I don't remember exactly but once I clicked on Enable for AngularJS Inspector extension and then I was able to use Augry. So, this could be a conflict with another extension because some extensions change DOM. 

P.S. Thankfully, now I'm able to use VSCode's debugging features for Angular.
